lib/OperatingSystems/FreeBSD.php

```diff
@@ -53,8 +53,8 @@ public function getMemory(): Memory {
 
 		$result = preg_match_all($pattern, $swapinfo, $matches);
 		if ($result === 1) {
-			$data->setSwapTotal((int)$matches['Avail'][0]);
-			$data->setSwapFree($data->getSwapTotal() - (int)$matches['Used'][0]);
+			$data->setSwapTotal((int)($matches['Avail'][0] / 1024));
```

Comment:
swapinfo -k returns the values as KB. The conversion to bytes was missing here. Now they are correct.